tmt/convert.py

```diff
@@ -179,6 +179,7 @@ def read(path, makefile, nitrate, purpose, disabled):
     data for individual testcases (if multiple nitrate testcases found).
     """
 
+    global makefile_contact
```

Comment:
Not really fan of `global`, and I am not sure it's even needed here, it's only copy of data['contact'] anyway, can't you just use that?

Reply:
@hegerj I wasn't happy about the `global` variable as well, but I need to use the value of this variable later in another function, which I wasn't able to get there in another way. For now, it's the best solution I could have come with. 
The different (imo ugly) solution I'm thinking of now is that I could change the input of functions (for getting data from Nitrate) by adding one more input variable but I consider it as a big "hit" to the code.

Reply:
Do you really need it though? It has the same content as `data['contact']`, which is already passed as an argument to the `read_nitrate_case()`. Why don't just use that variable?

Reply:
As for now `read_nitrate_case()` does not get [data](https://github.com/psss/tmt/blob/master/tmt/convert.py#L480), but it is already passed to `read_nitrate()` as `common_data`. I think it's completely fine to hand over `common_data` to `read_nitrate_case()` as well. Better than using a global variable.

Reply:
Yeah I am sorry, I was wrong about the `data`.
I agree with Petr, you should use `common_data`.

Reply:
I've fixed that and pushed.

Reply:
Thanks!